.github/workflows/unittest.yaml

```diff
@@ -0,0 +1,36 @@
+name: unittest
+on: [push, pull_request]
+
+jobs:
+  unittest:
+    name: unittest
+    strategy:
+      matrix:
+        zk-version: [3.4.14, 3.5.6]
```

Comment:
Can we have an "automatic" version to follow release branches (something like "1.x", "tip") etc? It would be nice if new go/zk version get picked up when they (pre?)released.

Reply:
We are manually downloading these in the Makefile.  We could look at something more automatic but it will be custom written.